### PR TITLE
Added mini-toc to changelogs

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation task: https://mattermost.atlassian.net/browse/MM-36605

Updated:
Added mini TOCs to the following topics:
- Install > Install MM > Desktop App > Desktop Application Changelog
- Install > Install MM > MM Server > MM Changelog
- Install > Install MM > Mobile App > MM Mobile Apps Changelog
- Manage > Manage MM > Cloud Workspaces > MM Cloud Changelog

### CODE REVIEW

**Comment by hmhealey** [comment]
Do we need to have every single version in the table of contents for those pages? That seems like a lot to include at the top of the page. 0/5 on that though.

**Comment by cwarnermm** [comment]
@hmhealey Very fair point. In 3 of the 4 cases, it's easy to short those lists down. (For the fourth case -- desktop changelog -- I can move older, legacy releases under a catchall heading).

Can you suggest a cut off per changelog, please?

**Comment by hmhealey** [comment]
For the web app one, everything back to the last LTS release would probably make sense. I think that'd limit us to at most the last 6 releases since I'm pretty sure we do those twice a year. That's probably a decent number to use elsewhere as well.

**Comment by cwarnermm** [comment]
Thanks so much, @hmhealey. I'll trim those lists back to reasonable lengths. Great feedback!

File: source/help/apps/desktop-changelog.rst [hunk 1]
```diff
@@ -1,8 +1,17 @@
 Desktop Application Changelog
 ========================================
 
+Latest Mattermost Desktop App releases:
+
+- `Release v4.7 <#id2>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:6]:
```suggestion
- `Release v4.7 <#id1>`_
```

File: source/help/apps/desktop-changelog.rst [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+- `Release v4.6 <#id5>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:7]:
```suggestion
- `Release v4.6 <#id4>`_
```

File: source/help/apps/desktop-changelog.rst [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+- `Release v4.5 <#id17>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:8]:
```suggestion
- `Release v4.5 <#id16>`_
```

File: source/help/apps/desktop-changelog.rst [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
+- `Release v4.4 <#id33>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:9]:
```suggestion
- `Release v4.4 <#id32>`_
```

File: source/help/apps/desktop-changelog.rst [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
+- `Release v4.3 <#id52>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:10]:
```suggestion
- `Release v4.3 <#id51>`_
```

File: source/help/apps/desktop-changelog.rst [hunk 6]
```diff
@@ -1,8 +1,17 @@
...
+- `Release v4.2.3 <#id69>`_
```
Comment (cwarnermm) [inline source/help/apps/desktop-changelog.rst:11]:
```suggestion
- `Release v4.2.3 <#id68>`_
```

**Review by hmhealey — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**